src/styles/stars.scss

```
@@ -0,0 +1,27 @@
+@import 'variables';
+
+.stars {
+  display: flex;
+  align-items: center;
+  padding-top: 7px;
+
+  &__star {
+    width: $stars-size;
+    height: $stars-size;
+    margin-right: 4px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    text-align: left;
+  }
+
+  .stars--1 &__star:nth-child(-n + 4) {
```

Consider using classes from the previous task
```suggestion
  .stars--1 &__star:nth-child(-n + 1),
  ... ,
  .stars--4 &__star:nth-child(-n + 4) {
```